webapi/src/main/scala/org/knora/webapi/messages/v1/responder/projectmessages/ProjectMessagesV1.scala

```diff
@@ -75,12 +74,18 @@ sealed trait ProjectsResponderRequestV1 extends KnoraRequestV1
 
 // Requests
 /**
-  * Get all information about all projects.
+  * Get all information about all projects. The response is in for of [[ProjectsResponseV1]]
```

Comment:
No. If you could please look at the whole comment :-) There is a type there. I have just changed both comments to make it clearer:

For ProjectsGetRequestV1:
```
Get all information about all projects in form of [[ProjectsResponseV1]]
```

For ProjectsGetV1:
```
Get all information about all projects in form of a sequence of [[ProjectInfoV1]].


```